src/spikeinterface/extractors/tests/test_whitematterrecordingextractor.py

```diff
@@ -57,3 +57,23 @@ def test_on_data():
     assert recording.get_sampling_frequency() == sampling_frequency
     assert recording.get_num_channels() == num_channels
     assert recording.get_duration() == 1.0
+
+
+def test_kwargs():
```

Comment:
Oh, I see, yes, that's our bad as reviewers:

Instead of adding tests for kwargs specifically can you add a pickling/serialization tests like this:

https://github.com/h-mayorquin/spikeinterface/blob/786541e3c89abcd0d9f4be0ca2478259da3cba52/src/spikeinterface/extractors/tests/common_tests.py#L90-L102

That should cover the kwargs.

We should add some base testing that is for all recordings that contains this too but this is for ourselves in another PR.

Reply:
> Instead of adding tests for kwargs specifically can you add a pickling/serialization tests like this:

Done!